.github/workflows/test-exploration.yml

```diff
@@ -0,0 +1,62 @@
+name: Test / Exploration
```

Comment:
As far as I can tell, it's in a separate file because we don't want to block PRs on Node 14 (or whatever other versions we want to test in the future)? I'd like to think of a clearer name...